services/auto-scan/cloudbuild.yaml

```diff
@@ -0,0 +1,48 @@
+steps:
+  - name: 'gcr.io/cloud-builders/docker'
+    id: start_testdb
+    entrypoint: /bin/sh
+    args: ["-c", "docker run --net cloudbuild --name testdb -d --rm -p 127.0.0.1:5432:5432 -e POSTGRES_USER=track_dmarc -e POSTGRES_PASSWORD=postgres postgres:alpine"]
+
+  - name: mikewilliamson/wait-for
+    id: wait_testdb
+    args: ['testdb:5432']
+
+  - name: 'gcr.io/cloud-builders/docker'
+    id: start_dbmigrate
+    entrypoint: /bin/sh
+    args: ["-c", "docker run --net cloudbuild --name dbmigrate -d --rm -e POSTGRES_USER=track_dmarc -e POSTGRES_PASSWORD=postgres -e DB_HOST=testdb gcr.io/track-compliance/services/db-migration"]
```

Comment:
I think this would do the same thing: 
```
- name: "gcr.io/track-compliance/services/db-migration"
    id: start_dbmigrate
    env: 
    - POSTGRES_USER=track_dmarc
    - POSTGRES_PASSWORD=postgres
    - DB_HOST=testdb
```